src/QmlControls/QGCButton.qml

```diff
@@ -7,7 +7,7 @@ import QGroundControl.ScreenTools
 
 Button {
     id:             control
-    hoverEnabled:   true
+    hoverEnabled:   !ScreenTools.isMobile
```

Comment:
Is there some reason for this? Mobile with a mouse?

Reply:
No no, this disables hover on mobile, so it doesn't have nothing to do with mouse on mobile

The issue is that for some reason in mobile the buttons keep "hovered" after you click them, so it creates a lot of inconsistencies around the UI. I am not sure of the "deep" reason, but in any case the concept of "hover" for touch based controls doesn't make sense, so I thought it would be good to disable hover on mobile. 

After this change, the buttons no longer keep "hovered" after clicking them in Android.

Reply:
Hmm, I had noticed that hover thing as well but hadn't gotten around to figure it out. So this is fine then. I think there are other place where hover is on in other controls. Can you search for that and make sure there aren't other places that should change as well.